Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -531,6 +531,16 @@ class BinaryOperation: Operation {
     }
 }
 
+/// Assigns a value to its left operand based on the value of its right operand.
+class AssignmentOperation: Operation {
```

Comment:
I'm not 100% happy about this name since at first sight I might assume this is just a `x = y`, but that's exactly what this *isn't*. How about we change it to `BinaryOperationAndReassignment` or something like that? That seems a bit more intuitive, as this class uses a `BinaryOperator` internally but also performs what `Reassign` does. WDYT?

Reply:
Tbf this was the hardest part of the PR: finding a name for the operation that caused least confusion :D
I like your idea, could we perhaps shorten it to `BinaryOperationAndReassign`?

Reply:
Sounds good, let's go with that!